README.md

```diff
@@ -0,0 +1,76 @@
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Fifa 2021 Pre Game Press Conference
+
+>In the process of creating this Conference website, I set up a new repository and prepared it for development using best practices (e.g. linters). I then build the mobile version of the website using my own images and texts. After I then deployed the home and about pages on github pages.
+Next I adapted the mobile version to the desktop version of the two pages and deployed on github.
+The speaker section is created dynamically using javascript. The conference page is responsive and views well on different screen sizes.
+
+## Loom preview of project
+https://www.loom.com/share/42bd513c69fa4dc9a5d3dcd47eb46732?sharedAppSource=personal_library
+
```

Comment:
- [ ] In the recorded video you are expected to highlight a piece of code that you find interesting, kindly record an additional video talking about this.

Reply:
okay I'll add that.